README.md

```diff
@@ -30,3 +30,23 @@ The project is still in the development
 .header-grid - styles the grid container that wraps header items. To change the order of the elements, edit this style: grid-template-areas: 'today arrows month year'
 
 ```
+## **Props**
+
+| Props | Description | Default | Type |
+| ------ | ----------- | ------- | ---- |
+| events | List of events to be displayed on the calendar | `mockEvents` | CalendarEventType[] |
```

Comment:
remove `mockEvents` from default as we won't have them when the calendar will be released.

And ClendarEventType[] tells nothing to me. Eather write the whole type here or somewhere above like section types